crates/goose-server/src/routes/recipe.rs

```diff
@@ -88,16 +88,7 @@ async fn create_recipe(
         request.messages.len()
     );
 
-    let error_response = CreateRecipeResponse {
```

Comment:
ah, so no need for all these now?

Reply:
yeah, Agent is no longer an Option<> on AppState, so these errors are gone